### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by matzew**
@evanshortiss please review, and enable Travis 

once merged, it would be good if we get this lib/version out to the wild!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - "0.10"
+  - "4.4.2"
+sudo: false
+before_install:
+  - npm install -g npm@2.13.5
```
Comment (evanshortiss) [inline .travis.yml:7]: @matzew any particular reason grunt is required here?

**Comment by evanshortiss** [comment]
Out to the wild? It's already on npm if that's what you mean? If not let me know the plan 👍

File: .travis.yml (hunk 1 above)
Comment (matzew) [inline .travis.yml:7]: No, I just did what all the other node things do... 

Should I remove ?

**Comment by matzew** [comment]
> Out to the wild? It's already on npm if that's what you mean? 

I mean get `0.2.0` of this thing out to npmjs

File: .travis.yml (hunk 1 above)
Comment (evanshortiss) [inline .travis.yml:7]: Yeah, this project doesn't need grunt 😄

Comment (evanshortiss) [inline .travis.yml:7]: No worries. Will merge. It won't affect anything and can be removed anytime.

**Comment by evanshortiss** [comment]
@matzew published 0.2.0 on npmjs 👍

**Pull request merged**